pkg/telemetry/rtt.go

```diff
@@ -0,0 +1,115 @@
+package telemetry
+
+import (
+	"math"
+	"time"
+
+	"github.com/pion/rtcp"
+)
+
+const (
+	NSUS                 = 1000
+	microssec      int64 = 1000000
+	kNtpJan1970Sec int64 = 2208988800
+)
+const kMaxCompactNtp uint32 = 0xFFFFFFFF
```

Comment:
maybe include all these constants in the above block.

Reply:
done